### ISSUE TITLE
Add direct routes to specific Account management modals in Wallet

### ISSUE THREAD

**Issue by josheleonard** [brave/brave-browser#22588]
## Description 

Adds routes to directly open a page to specific account management modals in wallet

## Expected result:

All tabs of `wallet/crypto/accounts/add-account` should have a corresponding URL

- add account modal: `/crypto/accounts/add-account/create/:accountTypeName?`
- import account modals: `/crypto/accounts/add-account/import/:accountTypeName?`
- hardware wallet import modals: `/crypto/accounts/add-account/hardware/:accountTypeName?`

**Comment by srirambv** [comment]
<table>
   <tr>
      <th>Brave</th>
      <th colspan="3">1.41.93 Chromium: 103.0.5060.114 (Official Build) (64-bit) </th>
   </tr>
   <tr>
      <th>Revision</th>
      <th colspan="3"><code>a1c2360c5b02a6d4d6ab33796ad8a268a6128226-refs/branch-heads/5060@{#1124}</code></th>
   </tr>
   <tr>
      <th>OS
      </th>
      <th>:ballot_box_with_check: Linux</th>
      <th>:ballot_box_with_check: Windows 11 Version 21H2
         <br/> (Build 22000.708)
      </th>
      <th>:ballot_box_with_check: macOS Version 12.0.1
         <br/> (Build 21C52)
      </th>
   </tr>
   <tr>
      <td colspan=4></td>
   <tr>
      <td colspan=4>
         <ul>
         <li>Verified steps from issue</li>
         <li>Verified direct routes to specific Account management modals in wallet work</li>
         <li>Verified manually typing the direct routes also work</li>
        </ul>
      </td>
   </tr>
   <tr>
      <td>
      </td>
      <td><video width=1200 src=https://user-images.githubusercontent.com/17010094/177589302-40acccee-25b4-48ba-9ec5-758771a1064f.mp4>
      </td>
      <td><video width=1000 src=https://user-images.githubusercontent.com/17010094/177589975-8ac1a785-32de-4ee0-ba41-a92f2e899e87.mp4>
      </td>
      <td><video width=1000 src=https://user-images.githubusercontent.com/17010094/177589778-7119bd01-9259-40f3-9c06-e24130922898.mov>
      </td>
   </tr>
</table>
